src/setup.py

```diff
@@ -33,6 +33,8 @@ def __init__(self, session):
             getConfigListEntry(_("User"), config_root.user),
             getConfigListEntry(_("Password"), config_root.password),
             getConfigListEntry(_("Sitemap"), config_root.sitemap),
+            getConfigListEntry(_("Data refresh *100ms"), config_root.refresh),
+            getConfigListEntry(_("Dimmer step"), config_root.dimmer),
```

Comment:
Both texts should have polish translations in "po" file. Use poedit or whatever tool you like to add these entries.

Reply:
Done